### ISSUE TITLE
KSDD和RSDD问题

### ISSUE THREAD

**Issue by xiaoming-678** [xiaozhen228/Bayes-PFL#16]
你好，为什么我再浮现这俩个数据集上的精度差了那么多

<img width="1192" height="533" alt="Im

<img width="973" height="520" alt="Image" src="https://github.com/user-attachments/assets/a80aa9dd-01cf-4705-9cd9-69ab53e8bbe6" />

age" src="https://github.com/user-attachments/assets/413bfdef-1f40-45a8-a54e-ea2c84907f53" />

**Comment by xiaozhen228** [comment]
@xiaoming-678  Hello, some normal samples in the test sets of certain datasets might have been randomly selected from the training sets, since the original test sets did not contain any normal samples. Therefore, I suggest trying again using the dataset we uploaded.  


KSDD2, RSDD, and DAGM datasets were post-processed by us, including operations such as random cropping. The original DAGM dataset was designed for weakly supervised defect segmentation, and thus its pixel-level annotations are imprecise elliptical labels. To make it suitable for anomaly segmentation, we manually re-annotated the dataset with precise pixel-level labels. Moreover, since the test sets of these datasets contain no normal samples, they are not directly suitable for anomaly classification. Therefore, when generating standardized datasets, we randomly selected an equal number of normal samples from the training set to match the number of abnormal samples in the test set for evaluation purposes. Due to the randomness involved, re-running ./dataset/make_dataset.py may result in different selections of normal samples. Therefore, we have also uploaded the version used in our paper for reference: [[DATA_three.zip]](https://drive.google.com/file/d/1JrTddHh2THBytMzv8O5Ru8AiRSJDHXIi/view?usp=drive_link).)

**Comment by xiaoming-678** [comment]
Thank you.
